libcaf_net/test/endpoint_manager.cpp

```diff
@@ -162,19 +166,23 @@ CAF_TEST(send and receive) {
   CAF_CHECK_EQUAL(mgr->init(), none);
   mpx->handle_updates();
   CAF_CHECK_EQUAL(mpx->num_socket_managers(), 2u);
-  CAF_CHECK_EQUAL(write(sockets.second, hello_manager.data(),
-                        hello_manager.size()),
+  CAF_CHECK_EQUAL(write(sockets.second,
+                        as_bytes(make_span(hello_manager.data(),
```

Comment:
There's no need to pass in `data()` and `size()` by hand:

```suggestion
                        as_bytes(make_span(hello_manager))
```